### PR TITLE
added queue and worker for deviation

### PR DESCRIPTION
**Pull request by KelvinThai**
# Description
This PR implements deviation worker and queue.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.

## Deployment

- [x] Should publish Docker image

### CODE REVIEW

File: fetcher/src/job/job.processor.ts
Comment (martinkersner) [inline fetcher/src/job/job.processor.ts:47]: This condition is actually not correct because `lastSubmission` and `submission` operate with an extra decimals that are part of adapter definition (e.g. https://config.orakl.network/adapter/ada-usdt.adapter.json)

How about using this already implemented function instead? 

https://github.com/Bisonai/orakl/blob/f4d2b8cdbe8d1d02165471044369f7027e9815ff/fetcher/src/job/job.utils.ts#L119-L153

**Comment by KelvinThai** [comment]
I fixed all the issues. Do you think we can merge this now @martinkersner

File: fetcher/src/job/job.processor.ts
Comment (martinkersner) [inline fetcher/src/job/job.processor.ts:54]: Most likely the decimal value should be 8, but the actual settings that defines it is inside of adapter file. You should be able to get it the same way as threshold for example.

**Review by martinkersner — APPROVED** [review]
There is one thing left we should fix before merging. The rest looks great! Approving now, please fix the decimals, and then merge.

**Pull request merged**
